pkg/model/piped.proto

```diff
@@ -33,8 +33,9 @@ message Piped {
     }
 
     enum ConnectionStatus {
-        ONLINE = 0;
-        OFFLINE = 1;
+        UNKNOWN = 0;
+        ONLINE = 1;
+        OFFLINE = 2;
```

Comment:
OK. It is still fine to update the index number because this enum is not being used yet.

Reply:
Yes, I confirmed that on the codebase 👍